File: UtilitiesVB/Solution Interfaces/IFileSystemFolderPaths.vb
Public Interface IFileSystemFolderPaths
    ReadOnly Property FldrAppData As String
    ReadOnly Property FldrFlow As String
    ReadOnly Property FldrMyD As String
    ReadOnly Property FldrPreReads As String
    ReadOnly Property FldrRoot As String
    ReadOnly Property FldrStaging As String
    ReadOnly Property FldrPythonStaging As String
    Sub Reload()
    ReadOnly Property Filenames As IAppStagingFilenames
End Interface

File: UtilitiesVB/Solution Interfaces/IFileSystemFolderPaths.cs

namespace UtilitiesVB
{
    public interface IFileSystemFolderPaths
    {
        string FldrAppData { get; }
        string FldrFlow { get; }
        string FldrMyD { get; }
        string FldrPreReads { get; }
        string FldrRoot { get; }
        string FldrStaging { get; }
        string FldrPythonStaging { get; }
        void Reload();
        IAppStagingFilenames Filenames { get; }
    }
}